scripts/BuildLibsRelease.sh

```diff
@@ -1,113 +1,157 @@
-sudo mkdir -p /opt/local/lib
-sudo mkdir -p /opt/local/bin
+#!/bin/bash -e
 
-#setup permissions on /opt/local
-sudo chgrp -R staff /opt/local*
-sudo chmod -R g+w /opt/local*
+# System dependency checks...
+if [ ! -d /opt/local/lib ] || [ ! -d /opt/local/libdbg ] || [ ! -d /opt/local/bin ] || [ ! -d /opt/local/bin ] ; then
```

Comment:
if you're restricting to bash, more reliable to use `[[` instead of test

Reply:
Fair, will refactor.